clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala

```diff
@@ -12,22 +16,25 @@ class Exporter(spark : SparkSession, apiClient: ApiClient, repoName: String, dst
   def exportAllFromCommit(commitID: String): Unit = {
     val ns = apiClient.getStorageNamespace(repoName)
     val df = LakeFSContext.newDF(spark, repoName, commitID)
-    df.createOrReplaceTempView("commit")
+
+    val tableName = Random.alphanumeric.dropWhile(_.isDigit).take(8).mkString("") + "_commit"
+    df.createOrReplaceTempView(tableName)
 
     // pin Exporter field to avoid serialization
     val dst = dstRoot
-    val actionsDF = spark.sql("SELECT 'copy' as action, * FROM commit")
+    val actionsDF = spark.sql(s"SELECT 'copy' as action, * FROM ${tableName}")
 
-    export(ns, dst, actionsDF)
-    spark.sparkContext.stop()
+    export(ns, dst, actionsDF, false)
+    export(ns, dst, actionsDF, true)
```

Comment:
:warning: Must verify that if the first export fails, the second does not go through.